compose-method/README.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: pattern
+title: Compose-method
+folder: compose-method
+permalink: /patterns/compose-method/
+categories: Structural
+tags:
+ - Gang of Four
+---
+
+## Intent
+
+Transform the logic into a small number of intention-revealing steps at the same level of detail.
+
+## Explanation
+
+Real world example
+
+> to make the method easy to read.
+
+In plain words
+
+> Composed Method is a small, simple method that you can understand in seconds.
+
+Wikipedia says
+
+> In software engineering, composed method is to divide your program into methods that perform one identifiable task. Keep all of the operations in a method at the same level of abstraction. This will naturally result in programs with many small methods, each a few lines long.
+
+**Programmatic Example**
+
+Refactoring to Patterns: Simplification | Compose Method | InformIT
```

Comment:
Is it referring to the source of the example? Maybe add a link here to make it really easy for the user.

Reply:
Maybe we could also explain that in the example we refactor a complex method into the composed method pattern